packages/dev/docs/pages/react-stately/index.mdx

```diff
@@ -33,9 +34,9 @@ description: A library of React Hooks that provides cross-platform state managem
 A library of React Hooks that provides cross-platform state management for your design system.
 
 <p className={clsx(typographyStyles['spectrum-Body2'], styles.homeLinks)}>
-  <a href="getting-started.html">Get started</a>
+  <a href="getting-started.html" className={clsx(linkStyle['spectrum-Link'], styles.link)}>Get started</a>
```

Comment:
MDX v2 no longer seems to apply the components mapping to explicit elements like this, only to markdown elements (e.g. `[some link](url)`).

Reply:
@devongovett do let mdx folks know if this is weird, or kinda makes sense. I’m not 100% on this change.

Reply:
@wooorm 👋. I think it _kinda_ makes sense? It seems MDX v2 is moving more toward JSX and away from Markdown (e.g. comment syntax as well). In JSX the way to solve this is using components, rather than overriding how HTML elements are rendered. It was a convenient feature, but certainly not required.